storage/s3-sdk/src/main/java/ListGcsBuckets.java

```diff
@@ -25,17 +25,20 @@
 import java.util.List;
 
 public class ListGcsBuckets {
-  public static List<Bucket> listGcsBuckets(String googleAccessKeyId, 
+  public static void listGcsBuckets(String googleAccessKeyId,
       String googleAccessKeySecret) {
+
+    // String googleAccessKeyId = "GOOGTS7C7FUP3AIRVJTE2BCD";
```

Comment:
Are these real keys? Maybe something like this would make it more clear they need to replace with there own: 
```suggestion
    // String googleAccessKeyId = "your-google-access-key-id";
```

Reply:
They're close but not active/usable. I thought it made sense to show what they look like rather than their name.